### ISSUE TITLE
Move Gst* to Wine and add gst-libav

### ISSUE THREAD

**Issue by zeckma-tech** [glfs-book/glfs#50]
GLFS probably won't have another package category that has packages that depend on Gst*, so let's move it to Wine. Another thing is some apps/games when ran through Wine use audio formats like WMA which aren't covered by the current Gstreamer plugins in the book, but instead gst-libav, which in turn depends on FFmpeg. WMA support in FFmpeg doesn't need any extra dependencies, but we should still provide a solid base for FFmpeg. It helps Gstreamer in general, in turn.

TL;DR:
- Move Gst* to Wine chapter (X)
- Add solid deps of FFmpeg (X)
- Add FFmpeg (X)
- Adjust Gst* to link to new in-book dependencies (X)
- Add gst-libav (include note about how some apps/games need WMA decoding support) (X)
